fs/config/config.go

```diff
@@ -822,6 +822,30 @@ func Choose(what string, defaults, help []string, newOk bool) string {
 	}
 }
 
+// ChooseAnswer asks the user to enter yes or no
```

Comment:
Apart from the `what` string this does the same as the `Confirm` function doesn't it?

https://github.com/rclone/rclone/blob/243a868a5b826ee385d4b22d0a8a27a6b034b462/fs/config/config.go#L726-L729